### ISSUE TITLE
Questions about what's going on with Open-Assistant? please watch.

### ISSUE THREAD

**Issue by camsdixon1** [LAION-AI/Open-Assistant#3720]
https://youtu.be/gqtmUHhaplo

**Comment by RackYican** [comment]
idk

**Comment by aggregate1166877** [comment]
The TL;DR:

* Open-Assistant, in its current state, is meant to serve as a template for other AIs to be built upon.
* Its training data, unlike other AI datasets, does not contain copyrighted content. Every piece of training information was submitted with 100% consent, making it the most ethical dataset in existence. If you build a model with LAION's OA dataset, no one can accuse you of "stealing" from others.
* It clearly documents how it works. When OA started, the community mostly speculated about the inner workings of larger AI models. Now, we have a working examples not only from OA, but also other academic sources.
* While the project could have continued, a large chunk of contributors, after already doing significant amounts of work, lost interest and left the project. Thus, it makes sense to stop OA here as a working template. The OA project achieved its core goals and can now be the seed for the next generation of ethical AI.
